src/message.rs

```diff
@@ -29,6 +29,7 @@ pub trait Message: Debug + Send + Sync {
         tag: u32,
         wire_type: WireType,
         buf: &mut B,
+        ctx: DecodeContext,
```

Comment:
And `merge_field`? It is part of the `Message` trait, but it is also `doc(hidden)` and the comment suggests it should not be used by users.

Reply:
Right, this is not part of the public API, so this addition is fine.

Reply:
In reality this may still end up needing to be released as a minor version bump so as to avoid cargo resolution issues, but I wouldn't feel the need to release note the change.